src/lib/formatter/RuleResultRecombinator.ts

```diff
@@ -1,12 +1,12 @@
 import {SfdxError} from '@salesforce/core';
 import * as path from 'path';
-import {EngineExecutionSummary, RecombinedRuleResults, RuleResult, RuleViolation} from '../../types';
+import {EngineExecutionSummary, RecombinedData, RecombinedRuleResults, RuleResult, RuleViolation} from '../../types';
 import {OUTPUT_FORMAT} from '../RuleManager';
 import * as wrap from 'word-wrap';
 import {FileHandler} from '../util/FileHandler';
 import * as Mustache from 'mustache';
 import htmlEscaper = require('html-escaper');
-import * as csvStringify from 'csv-stringify';
+import { stringify } from 'csv-stringify';
```

Comment:
The modern version of `csv-stringify` has different exports than the older one we were using. So this had to change.